Dockerfile_build_ubuntu32

```diff
@@ -1,4 +1,4 @@
-FROM 32bit/ubuntu:14.04
+FROM 32bit/ubuntu:16.04
```

Comment:
I changed this to `ioft/i386-ubuntu:14.04` and it will be merged to master in #7248. Can we get that merged and then this can be rebased on top of that?